homeassistant/components/ld2410_ble/__init__.py

```diff
@@ -0,0 +1,119 @@
+"""The LD2410 BLE integration."""
+
+import logging
+
+from bleak_retry_connector import BleakNotFoundError
+from ld2410_ble import LD2410BLE, LD2410BLEState, get_device
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth.match import ADDRESS, BluetoothCallbackMatcher
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS, EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.core import Event, HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+PLATFORMS: list[Platform] = [Platform.BINARY_SENSOR]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up LD2410 BLE from a config entry."""
+    address: str = entry.data[CONF_ADDRESS]
+    ble_device = bluetooth.async_ble_device_from_address(
+        hass, address.upper(), True
+    ) or await get_device(address)
+    if not ble_device:
+        raise ConfigEntryNotReady(
+            f"Could not find LD2410B device with address {address}"
+        )
+    ld2410_ble = LD2410BLE(ble_device)
+
+    coordinator = LD2410BLECoordinator(hass, ld2410_ble)
+
+    try:
+        await ld2410_ble.initialise()
+    except BleakNotFoundError as exc:
```

Comment:
You should probably trap any `BleakError` here

Reply:
Good idea. Done.